JS-do-not-construct-regex-from-userinput/majorbyte_rebbl_clan.js

```
@@ -0,0 +1,576 @@
+        };
+      };
+
+      let returnValue = data.map(x => {
+        return {
+          competitionId : Number(x.Row.Id.Value.replace(/\D/g,"")),
+          competitionName: x.Row.Name,
+          coaches: x.participants.map(mapData).concat(x.tickets.map(mapData))
+        };
+      });
+
+      res.json(returnValue);
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)